presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -107,6 +125,65 @@ Property Name                                      Description
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
 
 ``hive.s3.sse.enabled``                            Enable S3 server-side encryption.                            ``false``
+
+``hive.metastore.authentication.type``             Hive metastore authentication type.                          ``SIMPLE``
+                                                   Possible values are ``SIMPLE`` or ``KERBEROS``.
+
+``hive.metastore.service.principal``               Hive metastore service principal.
+                                                   The ``_HOST`` placeholder is allowed here and it is
+                                                   substituted with the actual metastore host. Use ``_HOST``
+                                                   placeholder for the configurations with more that
+                                                   one Hive metastore servers.
+                                                   Example: ``hive/hive-server-host@EXAMPLE.COM`` or
+                                                   ``hive/_HOST@EXAMPLE.COM``.
+
+``hive.metastore.client.principal``                Hive metastore client principal.
+                                                   The ``_HOST`` placeholder is allowed here and it is
+                                                   substituted with the actual Presto server host. Use
+                                                   ``_HOST`` placeholder for the principal per server
+                                                   configurations.
+                                                   Example: ``presto/presto-server-node@EXAMPLE.COM`` or
+                                                   ``presto/_HOST@EXAMPLE.COM``.
+
+                                                   .. note::
+
+                                                        Deletion of the internal table data is the
```

Comment:
Suggest:

The principal specified by ``hive.metastore.client.principal`` must have sufficient privileges to remove files and directories within the ``hive/warehouse`` directory. If the principal does not, only the metadata will be removed, and the data will continue to consume disk space.

This occurs because the Hive metastore is responsible for deleting the internal table data. When the metastore is configured to use Kerberos authentication, all of the HDFS operations performed by the metastore are impersonated. Errors deleting data are silently ignored.